monai/data/utils.py

```diff
@@ -1370,7 +1370,7 @@ def json_hashing(item) -> bytes:
 
     """
     # TODO: Find way to hash transforms content as part of the cache
-    cache_key = hashlib.md5(json.dumps(item, sort_keys=True).encode("utf-8")).hexdigest()
+    cache_key = hashlib.md5(json.dumps(item, sort_keys=True).encode("utf-8"), usedforsecurity=False).hexdigest()
```

Comment:
Thanks, this requires python 3.9+, so maybe add a switch here based on sys.version_info.minor for 3.8 compatibility

Reply:
Added, lmk if you want a different format